src/App.js

```diff
@@ -25,38 +36,63 @@ const App = () => {
   };
 
   const goToStatic = () => {
-    history.push(`/static/`);
+    history.push('/static/');
     setIndexEdit(-1);
   };
 
+  const deleteTaskAll = () => {
+    axios.delete('http://localhost:8000/deleteTaskAll').then(
+      (res) => {
```

Comment:
res =>  setAllList([...res.data.data]));

Reply:
if need that `(res) => setAllList([...res.data.data])` - is fixed